i18n/en.json

```diff
@@ -1,9 +1,23 @@
 {
-	"@metadata": {
-		"authors": [
-			"Christian Dullweber",
-			"Katie Filbert"
-		]
-	},
-	"propertysuggester-desc": "Suggests properties when adding Wikibase statements."
+    "@metadata": {
+        "authors": [
+            "Christian Dullweber",
+            "Katie Filbert"
+        ]
+    },
+    "propertysuggester-desc": "Suggests properties when adding Wikibase statements.",
+    "apihelp-wbsgetsuggestions-description": "API module to get property suggestions for Wikibase entities.",
```

Comment:
Technically yes, but practically? Isn't "these are missing, consider adding these" the only use of these suggestions?